nuitka/plugins/PluginBase.py

```diff
@@ -910,3 +918,23 @@ def wrapped(*args, **kwargs):
                 return None
 
     return wrapped
+
+
+class TagContext(dict):
+    def __init__(self, full_name, *args, **kwargs):
+        self.full_name = full_name
+        dict.__init__(*args, **kwargs)
+
+    def __getitem__(self, key):
+        try:
+            return dict.__getitem__(self, key)
+
+        except KeyError:
+            if key.startswith("use_"):
+                self.info("Use default 'False' for not existing tag '%s'." % key)
+                return False
+
+            Tracing.plugins_logger.sysexit(
```

Comment:
This should be `self.sysexit` so the using plugin becomes part of the trace automatically.